src/pages/ussf-documentation.tsx

```diff
@@ -1,83 +1,59 @@
 import { useUser } from 'hooks/useUser'
 import { withDefaultLayout } from 'layout/DefaultLayout/DefaultLayout'
 import styles from 'styles/pages/ussf-documentation.module.scss'
 import Loader from 'components/Loader/Loader'
+import { GET_DOCUMENTS_PAGE } from 'operations/cms/queries/getDocumentsPage'
+import { DocumentType, DocumentSectionType, DocumentPageType } from 'types'
 
-const USSFDocumentation = () => {
+const USSFDocumentation = ({
+  documentsPage,
+}: InferGetServerSidePropsType<typeof getServerSideProps>) => {
   const { user } = useUser()
-
-  const testItems: AccordionItemProps[] = [
-    {
-      title: 'Essential Reading',
-      content: (
-        <div className={styles.accordionContent}>
-          <LinkTo
-            target="_blank"
-            rel="noreferrer noopener"
-            href="https://www.spaceforce.mil/Portals/1/Space%20Capstone%20Publication_10%20Aug%202020.pdf">
-            Space Capstone Publication: Spacepower. Doctrine for Space Forces
-          </LinkTo>
-
-          <LinkTo
-            target="_blank"
-            rel="noreferrer noopener"
-            href="https://media.defense.gov/2020/Nov/09/2002531998/-1/-1/0/CSO%20PLANNING%20GUIDANCE.PDF">
-            CSO’s Planning Guidance
-          </LinkTo>
-
-          <LinkTo
-            target="_blank"
-            rel="noreferrer noopener"
-            href="/uploads/Guardian Ideal - FINAL - 1600 17Sept21.pdf">
-            Guardian Ideal
-          </LinkTo>
-
-          <LinkTo
-            target="_blank"
-            rel="noreferrer noopener"
-            href="/uploads/US Space Force Enlisted Rank Insig Info Sheet (1).pdf">
-            USSF Enlisted Rank and Insignia
-          </LinkTo>
-
-          <LinkTo
-            target="_blank"
-            rel="noreferrer noopener"
-            href="/uploads/USSF Health Wellness and Fitness Memo dated 16Mar22.pdf">
-            USSF/S1 Health, Wellness and Fitness Memo (16 MAR 2022)
-          </LinkTo>
-
-          <LinkTo
-            target="_blank"
-            rel="noreferrer noopener"
-            href="/uploads/USSF Guardian Commitment Poster.pdf">
-            USSF Guardian Commitment Poster
-          </LinkTo>
-        </div>
-      ),
-      expanded: true,
-      id: 'essential-reading',
-      headingLevel: 'h3',
-    },
-  ]
+  const sections = documentsPage.sections
 
   return !user ? (
     <Loader />
   ) : (
     <div>
-      <h2>Official USSF documentation</h2>
+      <h2>{documentsPage.pageTitle}</h2>
       <Grid row gap="lg">
-        <Grid col={8}>
-          <Accordion
-            bordered={true}
-            items={testItems}
-            className={styles.accordion}
-          />
-        </Grid>
+        {sections &&
+          sections.map((s: DocumentSectionType) => (
+            <Grid col={8} className={styles.accordion} key={s.id}>
+              <Accordion
+                bordered
+                items={[
+                  {
+                    title: s.title,
```

Comment:
Dark mode styles weren't showing properly here. I moved `className={styles.accordion}` to the `Accordion` and that fixed it!
```suggestion
            <Grid col={8} key={s.id}>
              <Accordion
                className={styles.accordion}
                bordered
                items={[
                  {
                    title: s.title,
```

Reply:
thank you!